src/js/player.js

```diff
@@ -312,6 +312,44 @@ class Player extends Component {
     // Run base component initializing with new options
     super(null, options, ready);
 
+    // we don't want a source change for the first source set
+    let initialSourceSet = false;
+
+    if (tag.src) {
+      initialSourceSet = true;
+    }
+    const triggerSourceChange = () => {
+      if (!initialSourceSet) {
+        initialSourceSet = true;
+        return;
+      }
+      this.trigger('sourcechange');
+    };
+
+    if (window.MutationObserver) {
+      this.videoSrcObserver_ = new window.MutationObserver(triggerSourceChange);
+      this.videoSrcObserver_.observe(tag, {attributes: true, attributeFilter: ['src']});
+    } else {
+      Object.defineProperty(tag, 'src', {
```

Comment:
🤔  I'm really not a fan of modifying built-ins like this...

Reply:
Well we could just skip that part, and only do `sourcechange` for browsers that support `MutationObserver`, but I think that would limit the usefulness of this feature.